matlab/library/WBToolboxLibrary_repository.mdl

```diff
@@ -5934,12 +5932,111 @@ Library {
 	    }
 	  }
 	}
+	Block {
+	  BlockType		  S-Function
+	  Name			  "YarpRpc"
+	  SID			  "1848"
+	  Ports			  [1]
+	  Position		  [-70, 105, 50, 135]
+	  ZOrder		  252
+	  FunctionName		  "BlockFactory"
+	  Parameters		  "'YarpRpc','WBToolbox',risingEdgeTrigger,portName,rpcCommand"
+	  SFunctionDeploymentMode off
+	  EnableBusSupport	  off
+	  SFcnIsStateOwnerBlock	  off
+	  Object {
+	    $PropName		    "MaskObject"
+	    $ObjectID		    296
+	    $ClassName		    "Simulink.Mask"
+	    Type		    "YARP Rpc"
+	    Description		    "This block behaves as the command 'yarp rpc'. Accordingly, this is a blocked call blocking the "
+	    "simulation until a reply is received.\n\nIt takes two parameters:\n1) Rising edge trigger which is enabled by def"
+	    "ault. Rpc command is sent only if a rising edge is detected. i.e a signal change from 0 to 1. If rising edge trig"
+	    "ger is unchecked, a trigger signal of 1 will send the rpc command.\n2) Rpc server port name to which user wants t"
```

Comment:
Again, not blocking, but a common way to refer to this two variants of "activation" used in the PLC  world are "edge-trigger" and "level-triggered". Perhaps it could make sense to use them also in WB-Toolbox docs, if appropriate. See Section 1 of https://www.plcopen.org/sites/default/files/downloads/creating_plcopen_compliant_function_block_libraries.pdf for an example.

Reply:
> Again, not blocking,

@traversaro I am not following this clearly. If I understood well the rpc client call to rpc server is a blocking call right ? In that case the simulation will be blocked till you get a response, as mentioned in https://github.com/robotology/wb-toolbox/pull/178#discussion_r291093250

> common way to refer to this two variants of "activation" used in the PLC world are "edge-trigger" and "level-triggered".

I will update the block description with edge-triggered trigger as the default option and level-triggered as the other option if the edge-triggered is unchecked.

@diegoferigo

Reply:
> > Again, not blocking,
> 
> @traversaro I am not following this clearly. If I understood well the rpc client call to rpc server is a blocking call right ? In that case the simulation will be blocked till you get a response, as mentioned in [#178 (comment)](https://github.com/robotology/wb-toolbox/pull/178#discussion_r291093250)

I just meant that my comment was not blocking, I was not referring to the call.